python/paddle/fluid/layers/nn.py

```diff
@@ -864,12 +864,14 @@ def dynamic_gru(input,
                 is_reverse=False,
                 gate_activation='sigmoid',
                 candidate_activation='tanh',
-                h_0=None):
+                h_0=None,
+                origin_mode=True):
```

Comment:
To be backward compatible, will it is better to set the default value of `origin_mode` to be `False`